packages/desktop-client/src/components/reports/ModeButton.tsx

```diff
@@ -18,23 +20,25 @@ export function ModeButton({
 }: ModeButtonProps) {
   return (
     <Button
-      type="bare"
-      style={{
-        padding: '5px 10px',
-        backgroundColor: theme.menuBackground,
-        marginRight: 5,
-        fontSize: 'inherit',
-        ...style,
-        ...(selected && {
-          backgroundColor: theme.buttonPrimaryBackground,
-          color: theme.buttonPrimaryText,
-          ':hover': {
-            backgroundColor: theme.buttonPrimaryBackgroundHover,
-            color: theme.buttonPrimaryTextHover,
-          },
+      variant="bare"
+      className={String(
+        css({
+          padding: '5px 10px',
+          backgroundColor: theme.menuBackground,
+          marginRight: 5,
+          fontSize: 'inherit',
+          ...(selected && {
+            backgroundColor: theme.buttonPrimaryBackground,
+            color: theme.buttonPrimaryText,
+            ':hover': {
+              backgroundColor: theme.buttonPrimaryBackgroundHover,
+              color: theme.buttonPrimaryTextHover,
+            },
+          }),
+          ...style,
```

Comment:
This is the wrong place. Needs to match previous file. It's causing a regression in spending report.